src/status_im/ui/screens/home/views.cljs

```diff
@@ -178,14 +279,65 @@
         [react/view {:style               styles/counter-public
                      :accessibility-label :notifications-unread-badge}]])]))
 
+(views/defview qr-button []
+  [react/view {:style {:margin-left 12}}
+   [quo2.button/button {:style {:width 32 :height 32 :background-color (quo2.colors/theme-colors quo2.colors/ui-background-03-light quo2.colors/ui-background-03-dark) :border-radius 10}
```

Comment:
you don't have to use style, there is type and size , we should have all cases covered

Reply:
yes, for some super weird reason it was not working for me, I'll check that out and update it